debian/control

```diff
@@ -20,7 +20,6 @@ Architecture: any
 Multi-Arch: same
 Depends: libbson-dev,
          libmongocrypt0 (= ${binary:Version}),
-         libintelrdfpmath-dev (>= 2.0u2-6),
          ${misc:Depends}
```

Comment:
```suggestion
         libdfp-dev,
         ${misc:Depends}
```

We should specify `libdfp-dev` here, as things like headers, static libraries, and configure scripts could be needed by consumers of libmongocrypt.